flake.nix

```diff
@@ -44,7 +45,7 @@
         let
           pkgs = import nixpkgs ({
             inherit system;
-            overlays = [ self.overlays.${system} ];
```

Comment:
@ulrikstrid FYI this was causing an assertion to fail based on the recent work in https://github.com/NixOS/nixpkgs/pull/210019